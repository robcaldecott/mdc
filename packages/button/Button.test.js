import React from "react";
import { Button } from ".";
import renderer from "react-test-renderer";
import TestUtils from "react-dom/test-utils";

it("renders in the DOM", () => {
  TestUtils.renderIntoDocument(<Button>Button</Button>);
});

it("renders", () => {
  expect(
    renderer.create(
      <Button raised compact dense>
        Button
      </Button>
    )
  ).toMatchSnapshot();
});
