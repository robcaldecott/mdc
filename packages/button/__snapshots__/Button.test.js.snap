// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<button
  className="mdc-button mdc-button--raised mdc-button--compact mdc-button--dense"
>
  <i
    className="material-icons mdc-button__icon"
  >
    star
  </i>
  Button
</button>
`;
